converter/script.js

```diff
@@ -0,0 +1,23 @@
+function convert() {
+    var num = document.getElementById("input").value;
```

Comment:
the `convert()` function should act as a wrapper for the rest of your functionality, meaning, it should let functions inside it perform their specific actions without doing all that inside itself.

The conditional checking should be converted to a function that's something like `isCelsius(input)` or `isFahrenheit(input)`.

Reply:
ideally this function's body should look something like this:
```
var input = getInputValue();
var output = '';
if (isFahrenheit(input) {
  output = toFahrenheit(input)
} else if (isCelsius(input) {
  output = toCelsius(input)
}
updateResultElement(output)
```